arborio/neurolucida.cpp

```diff
@@ -781,7 +789,8 @@ asc_morphology parse_asc_string(const char* input) {
     return {stree, std::move(morphology), std::move(labels)};
 }
 
-ARB_ARBORIO_API asc_morphology load_asc(std::string filename) {
+
+inline std::string read_file(std::string filename) {
```

Comment:
Hey, I have the lingering feeling that this method is probably repeated all over the code base. 
If do not feel it's too much to pile on here, you could hunt these places down and move this here to something like `util::read_file_to_string`. Otherwise just leave a note, and we'll take care of it later.

Reply:
A quick scan of the codebase turned up very few instances. Ok!